commit_resource.txt

```diff
@@ -0,0 +1,3 @@
+This is a great website for writing quality commit messages.
+
+https://chris.beams.io/posts/git-commit/
```

Comment:
@vimusr very cool. I added this URL to the WIKI https://github.com/grayghostvisuals/practice-git/wiki/Resources